<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnAddLesson">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>730</y>
      <width>330</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnAddLesson{	
	background-color: rgb(74, 149, 223);
	color: white;
	border-radius:25px;
}
QPushButton#btnAddLesson:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(74, 149, 223), stop:1  rgb(74, 149, 223));
}
QPushButton#btnAddLesson:pressed{
	padding-left:2px;
	padding-top:2px;	
	background-color: rgb(74, 149, 223);
}</string>
    </property>
    <property name="text">
     <string>Kayıt Ekle</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>80</y>
      <width>331</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Kayıt Ekle&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuit">
    <property name="geometry">
     <rect>
      <x>1839</x>
      <y>40</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnQuit{	
	background-color: transparent;
	color: rgb(0, 0, 0);
	border-radius:5px;
}
QPushButton#btnQuit:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(255, 65, 65), stop:1  rgb(255, 65, 65));
}
QPushButton#btnQuit:pressed{
	padding-left:2px;
	padding-top:2px;	
	background-color: rgb(255, 65, 65);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/icons/feather/x.svg</normaloff>:/icons/feather/x.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineLessonName">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>180</y>
      <width>330</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border-bottom:2px solid rgba(105,118,132,255);

</string>
    </property>
    <property name="placeholderText">
     <string>Ders Adı</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineLessonCode">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>290</y>
      <width>330</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border-bottom:2px solid rgba(105,118,132,255);

</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Ders Kodu</string>
    </property>
   </widget>
   <widget class="QLabel" name="bg1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>4</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai Medium</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="title4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>390</y>
      <width>641</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Kayıtlı Dersler Listesi&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHome">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnHome{	
	background-color: rgb(74, 149, 223);
	color: white;
	border-radius:25px;
}
QPushButton#btnHome:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(74, 149, 223), stop:1  rgb(74, 149, 223));
}
QPushButton#btnHome:pressed{
	padding-left:2px;
	padding-top:2px;	
	background-color: rgb(74, 149, 223);
}</string>
    </property>
    <property name="text">
     <string>Anasayfa</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="listLessons">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>500</y>
      <width>761</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineLessonYear">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>400</y>
      <width>330</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border-bottom:2px solid rgba(105,118,132,255);

</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Ders Yılı</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineLessonAcademician">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>510</y>
      <width>330</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border-bottom:2px solid rgba(105,118,132,255);

</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Ders Ögretmeni</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineLessonBranch">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>620</y>
      <width>330</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border-bottom:2px solid rgba(105,118,132,255);

</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Sube</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>351</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/logo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
